src/GitHub.Api/IO/FileSystem.cs

```diff
@@ -242,6 +244,11 @@ public string GetCurrentDirectory()
             return Directory.GetCurrentDirectory();
         }
 
+        public string GetProcessDirectory()
+        {
+            return Directory.GetCurrentDirectory();
```

Comment:
Dont you mean to return `processDirectory` here?

Reply:
No, this is correct. The `processDirectory` field is not needed.

The process directory is whatever `Directory.GetCurrentDirectory()` returns, and our `GetCurrentDirectory()` "lies" about which directory is actually current. 

`FileSystem.GetCurrentDirectory` returns the repository root path, and `FileSystem.GetProcessDirectory` returns the Unity project path (which is what Unity sets the current directory to when it loads a project.

Most of the times these will be the same, but if the repository path is initialized somewhere else, we want NPath to always point to that as the current directory and operate based on the assumption that all paths start at the repository root (because that's where we always run our processes from).